Header header
# quaternions straight from IMU
float64 yaw
float64 pitch
float64 roll
# passed through KF
float64 northing
float64 easting
float64 height
#float64 northingVel
#float64 eastingVel
float64 bodyFrameVel
# straight from GPS
int8 no_satellites
string sol_status
